packages/common-integration-tests/src/storage/storage.ts

```diff
@@ -107,13 +103,14 @@ export function runStorageServiceTests(
   });
   describe('#store get put and delete', () => {
     it('put get and delete key in a store', async () => {
-      await WithStore(storageClient, testingStoreName, async () => {
+      const storeName = testStoreName();
+      await WithStore(storageClient, storeName, async () => {
```

Comment:
Creating and deleting stores is a fairly expensive operation, I'm worried that it might not be scalable to do that for every test. (Thoughts, @anp13 ?)

Zooming out a bit - what's the problem we're trying to solve that would preclude us from creating a single store to use for the whole test suite? potentially via `BeforeAll` and `AfterAll`?

Reply:
I agree with you @cprice404. I would avoid create/deleting a store unless you are actually testing the functionality of creation/deletion of a store. Also we have seen ddb tables left behind when we timeout and deleting the store does not clean it up in the case where it is still stuck in creating state. So best to cautious with these operations.

I think the issue here was that we were doing a get operation on a store which was deleted in the previous test case. But that should not be a problem if the delete store test case is the last one which it is now since you removed the 5s timeout retry test case.